README.md

```diff
@@ -11,19 +11,13 @@
 
 <div align="center" markdown="1">
 
-[![Python version](https://img.shields.io/badge/python-3.9%20%7C%203.10%20%7C%203.11%20%7C%203.12%20%7C%203.13-blue.svg)](https://pypi.org/project/vizro/)
-[![PyPI version](https://badge.fury.io/py/vizro.svg)](https://badge.fury.io/py/vizro)
-[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/mckinsey/vizro/blob/main/LICENSE.md)
-[![Documentation](https://readthedocs.org/projects/vizro/badge/?version=stable)](https://vizro.readthedocs.io/)
-[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/7858/badge)](https://www.bestpractices.dev/projects/7858)
+FIXME: maybe use two new lines between each [![Python version](https://img.shields.io/badge/python-3.9%20%7C%203.10%20%7C%203.11%20%7C%203.12%20%7C%203.13-blue.svg)](https://pypi.org/project/vizro/) [![PyPI version](https://badge.fury.io/py/vizro.svg)](https://badge.fury.io/py/vizro) [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/mckinsey/vizro/blob/main/LICENSE.md) [![Documentation](https://readthedocs.org/projects/vizro/badge/?version=stable)](https://vizro.readthedocs.io/) [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/7858/badge)](https://www.bestpractices.dev/projects/7858)
```

Comment:
@stichbury I don't massively mind these being wrapped but it is a bit unreadable compared to before. If it's possible to fix without changing how they actually render on the github page then that would be great. Otherwise it's ok how it is. I made the same comment in a few places but may well have missed some.

Reply:
What I'm unsure of @antonymilne is if we roll back what mdformat changed to have it how it was before, how do we prevent it from fixing it next run? Is there a flag we use to tell it to ignore a section?

Reply:
I don't think that's possible. You can ignore a whole file but not just sections unfortunately: https://github.com/executablebooks/mdformat/issues/53.